internal/v2/remote_signer/remote_signer_test.go

```diff
@@ -0,0 +1,41 @@
+package remotesigner
+
+import (
+	"testing"
+	"crypto"
+	"crypto/tls"
+	_ "embed"
+)
+
+var (
+	//go:embed example_cert_key/cert.pem
+	cert []byte
+	//go:embed example_cert_key/key.pem
+	key []byte
+)
+
+// TestNewSigner runs unit test for NewRemoteSigner.
+func TestNewSigner(t *testing.T) {
+	// Setup for static test.
+	cert0, err0 := tls.X509KeyPair(cert, key)
+	if err0 != nil {
+		t.Errorf("Test suite setup failed: %v", err0)
```

Comment:
nit: in Go, error messages should start with a lowercased word (unless it's a function name or some other proper name). Additionally, I would suggest adding a slightly more descriptive error message here, e.g. "failed to parse PEM cert and key"

Reply:
Will do, thanks for pointing this out!